integrations/Axeptio/browser.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+import makeACall from "./utils";
+
+class Axeptio {
+  constructor(config) {
+    this.clientId = config.clientId;
+    this.name = NAME;
+  }
+
+  loadScript() {
+    window.axeptioSettings = {
+      clientId: this.clientId,
+    };
+    (function (d, s) {
+      var t = d.getElementsByTagName(s)[0],
+        e = d.createElement(s);
+      e.async = true;
+      e.src = "//static.axept.io/sdk.js";
+      e.setAttribute("data-loader", LOAD_ORIGIN),
+        t.parentNode.insertBefore(e, t);
+    })(document, "script");
+  }
+
+  init() {
+    logger.debug("===In init Axeptio===");
+    this.loadScript();
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded Axeptio===");
+    return !!window.__axeptioSDK && typeof window.__axeptioSDK === "object";
+  }
+
+  isReady() {
+    logger.debug("===In isReady Axeptio===");
+    this.recordAxeptioEvents();
+    return !!window.__axeptioSDK;
+  }
+
+  // this fucntion is used to record the triggered axeptio events through callback
+  recordAxeptioEvents() {
```

Comment:
Are we not filtering out the events? I mean we are setting callbacks for all the events by default?

Reply:
Yeah, as we are not totally sure about the event's name. We haven't added any filters and the callback is set to all the triggered events. One check is there for the payload. If the payload is undefined we are only sending the event name else both.

Reply:
Okay let's support all the events later we can add some filters as an enhancement.